R/boundsPredprob.R

```diff
@@ -1,74 +1,86 @@
-#' Decision cutpoints for boundary (based on predictive probability)
+#' Decision cutpoints for boundary (based on predictive probability) for Decision 1 rule.
 #'
 #' This function is used to identify the efficacy boundary and futility
-#' boundary based on predictive probabilities, i.e.:
+#' boundary based on the following rules:
 #' Efficacy boundary: find minimum x (xU) where
-#' Pr(Pr(P > p | x, Y) >= tT | x) > phiU,
+#' Pr(Pr(P > p0 | x, Y, a, b) >= tT | x) >= phiU,
```

Comment:
I remember we had `RR` in the other PR instead of `P` here, would be good to make this consistent throughout this function's documentation